test/testutils/util.go

```diff
@@ -283,7 +283,7 @@ func SetLeaderPodToReady(ctx context.Context, k8sClient client.Client, podName s
 			return err
 		}
 
-		leaderPod.Labels[leaderworkerset.TemplateRevisionHashKey] = cr.Labels[leaderworkerset.TemplateRevisionHashKey]
+		leaderPod.Labels[leaderworkerset.RevisionKey] = revisionutils.GetRevisionKey(cr)
```

Comment:
```suggestion
		revisionutils.GetRevisionKey(leaderPod) = revisionutils.GetRevisionKey(cr)
```

Reply:
Doesn't make sense to use it here since we are setting the field, not retrieving the value

Reply:
yup, I was doing a sweep on all places where we use `leaderworkerset.RevisionKey`